### PR TITLE
Added input to storybook

### PR DESCRIPTION
**Pull request by mateuszs**
Added input to storybook

image
image

### CODE REVIEW

File: src/components/form-elements/Input.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  const onChange = event => {
+    setValue(event.target.value);
+  };
+
+  return { value, onChange };
+};
+
+export const Default = args => {
+  const props = useInput(args);
```
Comment (coderitual) [inline src/components/form-elements/Input.stories.jsx:22]: I am not sure why we need this hook.

File: src/components/form-elements/Input.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  return <Input {...args} {...props} />;
+};
+
+export const ErroredState = args => {
+  const props = useInput(args);
```
Comment (coderitual) [inline src/components/form-elements/Input.stories.jsx:28]: Like above. Do we need this hook?

File: src/components/form-elements/Input.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  return <Input {...args} {...props} />;
+};
+
+const args = {
+  value: "Some input value"
+};
+
+const argTypes = {
```
Comment (coderitual) [inline src/components/form-elements/Input.stories.jsx:37]: Could we please keep stories arguments close to story itself.

File: src/components/form-elements/Input.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  errorMessage: {
+    type: { name: "string", required: false }
+  }
+};
+
+Default.args = args;
+Default.argTypes = argTypes;
+
+ErroredState.args = { ...args, errorMessage: "Some error", invalid: true };
```
Comment (coderitual) [inline src/components/form-elements/Input.stories.jsx:49]: Could we please keep stories arguments close to story itself.

**Review by coderitual — CHANGES_REQUESTED** [review]
Thanks for the PR! Before we can go further can we please align on the desired story format?

File: src/components/form-elements/Input.stories.jsx (hunk 1 above)
Comment (mateuszs) [reply src/components/form-elements/Input.stories.jsx:22]: I thought we want to also edit this input from the input itself

**Review by coderitual — APPROVED** [review]
This looks good now! Thank you for the effort to follow guidelines.

**Pull request merged**